development/chromereload.js

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```

Comment:
This is a vendored file that we include in certain development builds - it is not meant to be run as an executable.

Reply:
Good catch, thanks! These are now resolved.